src/PSR7/SpecFinder.php

```diff
@@ -231,6 +231,8 @@ public function findResponseSpec($addr): ResponseSpec
      * @return Header[]
      *
      * @throws NoPath
+     *
+     * @phpstan-return array<string, Header|Parameter>
```

Comment:
AFAIK modern IDEs understand plain `@return` with the same contents as well, no need in special phpstan prefix. Can you check it ?

Reply:
Yeah, at least PHPStorm recognizes it correctly. Changed.